### PR TITLE
docs(plugins): remove all depreciated plugins

### LINKED ISSUE #3775: remove deprecated plugins from https://webpack.js.org/plugins/
__Feature to document__

Need to remove:

- i18n-webpack-plugin
- extract-text-webpack-plugin
- babel-minify-webpack-plugin
- uglifyjs-webpack-plugin
- zopfli-webpack-plugin

__Author(s)__

Any

__Additional information__

No

[ ] I am willing to work on this issue and submit a pull request.

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #3775

### CODE REVIEW

File: src/content/plugins/commons-chunk-plugin.md [hunk 1]
```diff
@@ -216,22 +217,6 @@ new webpack.optimize.CommonsChunkPlugin({
 });
 ```
 
-In order to obtain a single CSS file containing your application and vendor CSS, use the following `minChunks` function together with [`ExtractTextPlugin`](/plugins/extract-text-webpack-plugin/):
```
Comment (EugeneHlushko) [inline src/content/plugins/commons-chunk-plugin.md:219]: Maybe you want to add another PR to describe the way to achieve this with modern plugins/config?

**Comment by snitin315** [comment]
@EugeneHlushko Can you guide me for same?
This link seems to be inside `plugins/_mini-css-extract-plugin` which is being ignored by git?

File: src/content/plugins/commons-chunk-plugin.md (hunk 1 above)
Comment (snitin315) [reply src/content/plugins/commons-chunk-plugin.md:219]: Sure, looking into it.

**Comment by montogeek** [comment]
@snitin315 `dist/plugins/extract-text-webpack-plugin` doesn't exist anymore because this same PR deletes it.
Yes, file is ignored by git but still it is deployed, which mean you will need to remove it from mini-css-extract-plugin own README file.

**Comment by EugeneHlushko** [comment]
```
✖ FAIL load dist/plugins/extract-text-webpack-plugin
  | operator: load
  | expected: 200 dist/plugins/extract-text-webpack-plugin
  |   actual: ENOENT: no such file or directory, open '/home/travis/build/webpack/webpack.js.org/dist/plugins/extract-text-webpack-plugin/'
  |       at: dist/loaders/extract-loader/index.html:10:53 <a href="/plugins/extract-text-webpack-plugin/">...</a>
```

**Comment by snitin315** [comment]
Need to merge https://github.com/peerigon/extract-loader/pull/87

**Comment by evilebottnawi** [comment]
I'm afraid it will never be merged, what is out workaround?

**Comment by snitin315** [comment]
Need to investigate why our linting depends on other repositories readme files. /cc @montogeek

**Comment by montogeek** [comment]
We could skip this link in particular.

**Comment by evilebottnawi** [comment]
@montogeek Can you take care about it? Thanks!

**Comment by evilebottnawi** [comment]
@webpack/documentation-team Please start to solve this issue ASAP, I get questions and error messages every day and it wastes my time

**Comment by chenxsan** [comment]
@snitin315 Could you try to update `"lint:links"` script in `packages.json` to this?:

```json
"lint:links": "hyperlink -c 8 -r dist/index.html --canonicalroot https://webpack.js.org/ -i --skip /plugins/extract-text-webpack-plugin/ --skip https://img.shields.io --skip **/printable** --skip https://david-dm.org --skip https://codecov.io/gh --skip 'content-type-mismatch https://travis-ci.org' > internal-links.tap; cat internal-links.tap | tap-spot"
```
It fixed the CI failure in my test locally.

**Comment by snitin315** [comment]
Thanks @chenxsan I will update.

**Comment by chenxsan** [comment]
The current issue was gone as far as I see. Now there's another problem coming from source-map-loader:

```
  ✖ FAIL fragment-check dist/loaders/source-map-loader/index.html --> #url
  | operator: fragment-check
  | expected: id="url"
  |       at: dist/loaders/source-map-loader/index.html:1:14330 <a href="#url">...</a>
```
I've already submitted a pull request there https://github.com/webpack-contrib/source-map-loader/pull/126, hopefully we can get the CI fixed soon once it is merged.

**Comment by snitin315** [comment]
Can we skip all the `/plugins/**` & `/loaders/**` links?

**Comment by chenxsan** [comment]
@snitin315 CI should pass now if you rerun it.

**Comment by chenxsan** [comment]
I think `lint:links` script is good at catching those potential broken links. The problem here is we lint third-party plugins/loaders too, which might block CI if pull requests can't be merged there.

**Review by evilebottnawi — APPROVED** [review]
Thanks!

**Review by EugeneHlushko — APPROVED** [review]
Yup it works now, indeed its hard given that readmes are outside of our codebase in webpack.js.org repo, but serving broken links never works well, thats why it was decided to keep it at error level, nobody would care if it was a warning

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**